src/APIUtils/APIUtils.res

```diff
@@ -1189,6 +1189,8 @@ let useGetURL = () => {
       | INTEGRATION_DETAILS => `user/get_sandbox_integration_details`
       | SDK_PAYMENT => "payments"
       | CHAT_BOT => `chat/ai/data`
+      | FEE_OVERVIEW_ESTIMATE => `hypersense/fee_estimate/get-overview-estimate`
+      | FEE_TRANSACTION_ESTIMATE => `hypersense/fee_estimate/get-transaction-fee-estimation`
```

Comment:
can you declare a variable for hypersense base url, and if it is used at other places also update them